paasta_tools/cli/cmds/logs.py

```diff
@@ -559,6 +610,86 @@ def cluster_to_scribe_env(self, cluster):
             return env
 
 
+def generate_start_end_time(from_string="30m", to_string=None):
+    """
```

Comment:
sorry to nitpick, but can you fix the format of your docstrings please https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings and make sure you add ``:param:`` and ``:returns:`` docs